data-raw/query-urls.R

```diff
@@ -33,12 +33,23 @@ query_urls <-
+    "lookup", "lad_counties_21", "30.03.21", "OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LTLA21_UTLA21_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/lower-tier-local-authority-to-upper-tier-local-authority-april-2021-lookup-in-england-and-wales/explore",
+    "lookup", "msoa_lad_19", "18.12.19", "OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/MSOA11_WD19_LAD19_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::middle-layer-super-output-area-2011-to-ward-to-lad-december-2019-lookup-in-england-and-wales/about",
+    "lookup", "msoa_lad_20", "30.11.20", "OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/MSOA11_WD20_LAD20_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::middle-layer-super-output-area-2011-to-ward-to-lad-december-2020-lookup-in-england-and-wales/about",
+    "lookup", "msoa_lad_21", "25.11.21", "OGLv3", "https://www.arcgis.com/sharing/rest/content/items/b4c0247ef8564d419bdeb1eda13f2d0f/data" , "https://geoportal.statistics.gov.uk/documents/middle-layer-super-output-area-2011-to-ward-to-lad-december-2021-lookup-in-england-and-wales/about",
+    "lookup", "lsoa_lad_21", "25.11.21", "OGLv3", "https://www.arcgis.com/sharing/rest/content/items/81bcefcd048e43acb948ad069c5e06c0/data", "https://geoportal.statistics.gov.uk/documents/ons::lower-layer-super-output-area-2011-to-ward-2021-to-lad-2021-lookup-in-england-and-wales/about",
+    "lookup", "lsoa_msoa_11", "03.08.18", "OGLv3", "https://opendata.arcgis.com/datasets/fe6c55f0924b4734adf1cf7104a0173e_0.csv", "http://geoportal.statistics.gov.uk/datasets/fe6c55f0924b4734adf1cf7104a0173e_0",
+    "lookup", "postcode_oa_lsoa_msoa_lad_20", "23.11.20", "OGLv3", "https://www.arcgis.com/sharing/rest/content/items/9f629cd0c2fb42beb0d79a474fee4776/data", "https://geoportal.statistics.gov.uk/datasets/9f629cd0c2fb42beb0d79a474fee4776/about",
+    "lookup", "lad_cauth_20", "27.11.2020","OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LAD20_CAUTH20_EN_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::local-authority-district-to-combined-authority-december-2020-lookup-in-england/about",
+    "lookup", "lad_fra_21", "30.03.21","OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LAD21_FRA21_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::local-authority-district-to-fire-and-rescue-authority-april-2021-lookup-in-england-and-wales/about",
```

Comment:
Please rearrange lookups into alphabetical order for easier maintenance/reference

Reply:
Resolved in https://github.com/humaniverse/geographr/pull/24/commits/e1ceb0d88b1da644f9be739f026c2d73123fd31b.